examples/platforms/nrf52811/README.md

```diff
@@ -0,0 +1,221 @@
+   `/dev/ttyACM1` (for the second one).
+
+```bash
+$ /usr/bin/ot-cli /dev/ttyACM0 115200
+```
+
+Now you are connected with the CLI, commands below shoud be used to form a network:
+```bash
+ > panid 0xabcd
+ Done
+ > ifconfig up
+ Done
+ > thread start
+ Done
+```
+
+   
+3. Open a terminal connection on the first board and start a new Thread network.
+
+ ```bash
+ > panid 0xabcd
+ Done
+ > ifconfig up
+ Done
+ > thread start
+ Done
+ ```
+
+4. After a couple of seconds the node will become a Leader of the network.
+
+ ```bash
+ > state
+ Leader
+ ```
+
+2. The CLI MTD example uses the direct UART connection. To view raw UART output, start a terminal
+   emulator like screen and connect to the used COM port with the following UART settings:
+    - Baud rate: 115200
+    - 8 data bits
+    - 1 stop bit
+    - No parity
+    - HW flow control: RTS/CTS
+   Open a terminal connection on the second board and attach a node to the network.
+
+ ```bash
+ screen /dev/ttyACM1 115200
+ ```
+
+Now you are connected with the CLI, commands below shoud be used to form a network:
+
+ ```bash
+ > panid 0xabcd
+ Done
+ > ifconfig up
+ Done
+ > thread start
+ Done
+ ```
+
+6. After a couple of seconds the second node will attach and become a Child.
+
+ ```bash
+ > state
+ Child
+ ```
+
+7. List all IPv6 addresses of the first board.
+
+ ```bash
+ > ipaddr
+ fdde:ad00:beef:0:0:ff:fe00:fc00
+ fdde:ad00:beef:0:0:ff:fe00:9c00
+ fdde:ad00:beef:0:4bcb:73a5:7c28:318e
+ fe80:0:0:0:5c91:c61:b67c:271c
+ ```
+
+8. Choose one of them and send an ICMPv6 ping from the second board.
+
+ ```bash
+ > ping fdde:ad00:beef:0:0:ff:fe00:fc00
+ 16 bytes from fdde:ad00:beef:0:0:ff:fe00:fc00: icmp_seq=1 hlim=64 time=8ms
+ ```
+
+For a list of all available commands, visit [OpenThread CLI Reference README.md][CLI].
+
+[CLI]: ./../../../src/cli/README.md
+
+## Logging module
+
+By default, the OpenThread's logging module provides functions to output logging
+information over SEGGER's Real Time Transfer (RTT).
+
+RTT output can be viewed in the J-Link RTT Viewer, which is available from SEGGER.
+The viewer is also included in the nRF Tools. To read or write messages over RTT,
+connect an nRF5 development board via USB and run the J-Link RTT Viewer.
+
+Select the correct target device (nRF52) and the target interface "SWD".
+
+The intended log level can be set using `OPENTHREAD_CONFIG_LOG_LEVEL` define.
+
+## Disabling the Mass Storage Device
+
+Due to a known issue in Segger’s J-Link firmware, depending on your version, you might experience data corruption or drops if you use the serial port. You can avoid this issue by disabling the Mass Storage Device:
+
+ - On Linux or macOS (OS X), open JLinkExe from the terminal.
+ - On Microsoft Windows, open the J-Link Commander application.
+
+Run the following command: `MSDDisable`
+
+## Diagnostic module
+
+nRF52811 port extends [OpenThread Diagnostics Module][DIAG].
+
+You can read about all the features [here][nRFDIAG].
```

Comment:
Do not place links under "here". Rephrase to something like "For more information, see <title of the page>."

Reply:
fixed